mini-bank/mini-bank.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+/*Create account page*/
+.page-desc {
+  font-size: 40px;
+}
+.create-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: 15% auto;
+}
+.create-button {
+  font-size: 25px;
+  border-radius: 10px;
+}
+.create-button:hover {
+  background: #aaa;
+  color: #000;
+  transition: 0.8s;
+}
+/*Transaction page*/
+.header-component {
```

Comment:
Would be nice to make the navigation buttons wrap nicely on smaller screens, so they don't overflow the page container.